library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -3,8 +3,8 @@ use crate::fmt;
 use crate::io::{self, Error, ErrorKind};
 use crate::ptr;
 use crate::sys;
-use crate::sys::cvt;
 use crate::sys::process::process_common::*;
+use crate::sys::{cvt, cvt_nz};
```

Comment:
I think this will give an `unused` warning (denied) on some targets. Better to import it at the start of `posix_spawn` where you know it's used.